editor/src/clj/editor/scene.clj

```diff
@@ -850,7 +850,7 @@
               (do (ui/text! info-label info-text)
                   (ui/visible! info-label true))))
           (when (and (some? drawable) (some? async-copy-state-atom))
-            (update-image-view! image-view drawable async-copy-state-atom evaluation-context)))))))
+            (update-image-view! image-view drawable async-copy-state-atom evaluation-context dt)))))))
```

Comment:
Not sure about function arg order here. Is it ok to put dt last or do you always want the evaluation-context as the last arg?